src/java.base/share/classes/jdk/internal/logger/LoggerFinderLoader.java

```diff
@@ -117,6 +137,28 @@ private static Iterator<System.LoggerFinder> findLoggerFinderProviders() {
         return iterator;
     }
 
+    static class TemporaryLoggerFinder extends LoggerFinder {
+        private static final BiFunction<String, Module, Logger> loggerSupplier =
+                new BiFunction<>() {
+                    @Override
+                    public Logger apply(String name, Module module) {
+                        return LazyLoggers.getLoggerFromFinder(name, module);
```

Comment:
A better idea might be to:
```suggestion
                        return LazyLoggers.getLogger(name, module);
```